HeterogeneousCore/SonicTriton/test/tritonTest_cfg.py

```diff
@@ -62,6 +80,21 @@
                 modelName = cms.string(model),
                 modelVersion = cms.string(""),
                 modelConfigPath = cms.FileInPath("HeterogeneousCore/SonicTriton/data/models/{}/config.pbtxt".format(model)),
+                verbose = cms.untracked.bool(options.verbose or options.verboseClient),
+                useSharedMemory = cms.untracked.bool(not options.noShm),
+                compression = cms.untracked.string(options.compression),
```

Comment:
incorrect rebase. these lines are contained in https://github.com/cms-sw/cmssw/blob/79b590e4ef3ac7018721083e171b90f1d6b13e8f/HeterogeneousCore/SonicTriton/python/customize.py#L92
remove these lines